commands/accounts/delete.go

```diff
@@ -0,0 +1,78 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package accounts holds commands for accounts command.
+package accounts
+
+import (
+	"context"
+	"errors"
+	"net/url"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+
+	accountscli "github.com/percona/everest/pkg/accounts/cli"
+	"github.com/percona/everest/pkg/kubernetes"
+)
+
+// NewDeleteCmd returns a new delete command.
+func NewDeleteCmd(l *zap.SugaredLogger) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "delete",
+		Example: "everestctl accounts delete --username user1 --password $USER_PASS",
+		Run: func(cmd *cobra.Command, args []string) { //nolint:revive
+			initDeleteViperFlags(cmd)
+
+			kubeconfigPath := viper.GetString("kubeconfig")
+			username := viper.GetString("username")
+			password := viper.GetString("password")
```

Comment:
@mayankshah1607 Didn't we decide we'd be removing the authentication from this command?